applications/ShapeOptimizationApplication/python_scripts/algorithm_base.py

```diff
@@ -14,16 +14,20 @@
 
 # ==============================================================================
 class OptimizationAlgorithm:
+    # --------------------------------------------------------------------------
+    def CheckApplicability(self):
```

Comment:
`Check` is more consistent with Kratos style

Reply:
I would just avoid to introduce differences between this application and the rest of Kratos if there is no clear benefit. What else than applicability could you check?